dashboard/app/serializers/api/v1/pd/application_serializer.rb

```diff
@@ -167,6 +167,11 @@ def percent_string(count, total)
     "#{(100.0 * count / total).round(2)}%"
   end
 
+  def yes_no_string(value)
+    return nil unless value
+    value ? Pd::Application::ApplicationBase::YES : Pd::Application::ApplicationBase::NO
```

Comment:
I think this function will only return `YES` or `nil`; the `NO` case is unreachable.  Can we simplify it?

Reply:
this is an embarrassing bug. Will change the first line to `return nil if value.nil?`.
I'm using `ApplicationBase::YES|NO` instead of 'Yes|No' string to keep it consistent with values returned by teacher application `auto_score!` function.